### PR TITLE
add OpenShift Service Mesh option to the setup

### LINKED ISSUE #143: Using Maistra to install Istio
It will be nice to have Istio installed using[ Maistra.io ](http://maistra.io/)

### PR DESCRIPTION
**Pull request by tvieira**
This add a second option to install Istio using OpenShift Service Mesh installation method (via operator).

Fix #143

### CODE REVIEW

File: documentation/modules/ROOT/pages/1setup.adoc [hunk 1]
```diff
@@ -67,8 +68,136 @@ oc login $(minishift ip):8443 -u admin -p admin
...
+                    "kind": "WebhookAdmission",
+                    "kubeConfigFile": "/dev/null"
+                }
+            }
+        }
+    }
+}'
+----
+
+You also need to set the system limits on mmap counts otherwise Elastic Search will fail. Jaeger uses Elastic Search and it is part of the Reed Hat Istio bundle. Elasticsearch uses a mmapfs directory by default to store its indices. The default operating system limits on mmap counts is likely to be too low, which may result in out of memory exceptions. To fix this you have to login into the minishift instance and do the following
```
Comment (lordofthejars) [inline documentation/modules/ROOT/pages/1setup.adoc]: In big sentences like this add a break line after each period (.) char. In Asciidoc you need two line breaks to make a line break, so in that way is easy to review.

Reed -> Red

The official name is Elasticsearch, not Elastic Search.

I will add mmap, mmapfs between (`).

Comment (tvieira) [reply documentation/modules/ROOT/pages/1setup.adoc]: Thanks for the review! All fixed.

**Comment by lordofthejars** [comment]
Thank you very much for this contribution.

**Comment by tvieira** [comment]
@lordofthejars is there anything else I should do to get it merged?

**Comment by lordofthejars** [comment]
I'll do it this night, I was waiting for any other reviewers but since no
more comments, I'll do this night.

Sorry for taking so much time

**Pull request merged**